test/e2e/e2e_dm_test.go

```diff
@@ -58,6 +58,9 @@ func TestDMSubtask(t *testing.T) {
 
 	dmSubtask, err := ioutil.ReadFile("./dm-subtask.toml")
 	require.NoError(t, err)
+	// TODO: in #272, Server.jobManager is assigned after Server.leader.Store(), so even if we pass the PreRPC check,
+	// Server.jobManager may not be assigned yet. We simply sleep here
+	time.Sleep(time.Second)
```

Comment:
No need to address, maybe we can add one server master API to test server master leader is available in the feature.

Reply:
yes, will solve it in future